dags/incline_dash.py

```diff
@@ -41,7 +37,7 @@
         dataset_id=dataset,
         # We recreate this entire table from scratch every day because we are
         # taking the last seen migration ping over all time for each client.
-        destination_table=None,
+        destination_table="migrated_clients_v1",
```

Comment:
This isn't necessary, [the query creates the table itself.](https://github.com/mozilla/bigquery-etl/blob/master/sql/org_mozilla_firefox_derived/migrated_clients_v1/init.sql#L1)